bcgov/mds
# MDS-3848 - init build refactor for filesystem-provider

# Main

Refactoring build process for filesystem-provider service.

# Other
- N/A

# How to test

- Tag and build through oc tag command (ensure you're logged into the OC console)
- Check if the container is running
- Boom, done. 

# Notes

- N/A
